plotter.py

```diff
@@ -0,0 +1,29 @@
+"""
+Plotter functions.
+"""
+
+import pandas as pd
+import plotly
+import plotly.express as px
+
+def line(raw_data, tittle='2D-line',
+         x_axial='X', y_axial='Y', instruments='instruments'):
+    '''
+    Function for 2D-line.
+
+    Raw_data is a dict of tuple list.
+    The keys means names of data, and each tuple means a coordinate point
+    '''
+    plot_data = {
+        instruments : [],
+        x_axial : [],
+        y_axial : []
+    }
+    for instrument, values in raw_data.items():
+        for value in values:
+            plot_data[x_axial].append(value[0])
+            plot_data[y_axial].append(value[1])
+            plot_data[instruments].append(instrument)
+    data = pd.DataFrame(data=plot_data)
+    fig = px.line(data, x=x_axial, y=y_axial, color=instruments, title=tittle)
+    plotly.offline.plot(fig)
```

Comment:
`offline.plot` will trigger the browser and open the page, which may not be really useful in the future.
Why not return the generate html code directly?
This is doable by [plotly.io.to_html](https://plotly.com/python-api-reference/generated/plotly.io.to_html.html?highlight=offline#plotly-io-to-html).